02_Methodiek.Rmd

```diff
@@ -0,0 +1,374 @@
+
+
+Het al dan niet weerhouden van een tijdreeks kan gevolgen hebben voor de categorie-indeling van een rastercel. 
+Voor een rastercel kunnen immers na een tijdreeksanalyse meer of ook minder peilbuizen geschikt zijn. 
+Daarom voeren we na de tijdreeksanalyse de categorisering nog een tweede keer (en mogelijk nog meerdere keren) uit, waarbij met de uitkomst van de tijdreeksanalyse rekening wordt gehouden. **_of verkiezen we de peilbuizen met de beste modellen ?_**
+
+
+#### Bijkomende GRTS-analysen
+
+Afhankelijk van de verhouding tussen het aantal gewenste en beschikbare peilbuizen, moeten er nog enkele bijkomende GRTS-analysen uitgevoerd worden.
+
+##### Selecteren van potentiële geschikte habitatvlekken voor rastercellen waarvoor er te weinig peilbuizen bestaan (cat. 1) {#sel-pot-habvlek}
+
+Voor rastercellen van cat. 1 is er een nood aan één of meer extra meetpunten.
+Om hierbinnen (8192 * 8192 m) de plaatskeuze zo aselect mogelijk te doen, doen we beroep op het fijnste GRTS-raster 'GRTSmaster_habitats' (celgrootte = 32 m). 
+Opnieuw wordt op deze manier een synergie bekomen met de aselecte locatiekeuzes in MNM.
+Elk deelrastercel heeft binnen de rastercel een bepaalde rangnummer. 
+Alleen deelrastercellen _die voor het merendeel vegetaties bevatten die tot de GT-groep behoren_ komen voor selectie in aanmerking. 
+De deelrastercel(len) met het /de laagste rangnummer(s) zal/zullen dan geselecteerd worden. 
+Aan de selectie van deelrastercellen kunnen, naast de aanwezigheid van de juiste GT-groep, nog bijkomende voorwaarden gekoppeld worden (beheerder, ...).
+Blijkt bij inspectie op terrein dat de geselecteerde plaats toch niet aan de verwachtingen/criteria voldoet, dan wordt binnen dezelfde rastercel de rastercel die het eerst in rang volgt binnen een zoekstraal van 1 km bezocht.  
+
+
+##### Selecteren van een meetpunt in een rastercel waarvoor meerdere kandidaten zijn (cat. 3) {#sel-habvlek}
+Als het aantal beschikbare peilbuizen groter is dan het gewenste aantal en wanneer ook het aantal geschikte peilbuizen (cfr. tijdreeksanalyse) hiervan afwijkt, dient nog een bijkomende selectie van peilbuizen te gebeuren.
+Indien het aantal geschikte peilbuizen dan groter is dan het gewenste aantal wordt enkel binnen deze groep verder gezocht.
+In het andere geval omvat het alle peilbuizen.
+
+Net zoals voor cat. 1 willen we de keuze zo aselect mogelijk doen. 
+We doen hiervoor terug beroep op het fijnste GRTS-raster 'GRTSmaster_habitats' (celgrootte = 32 m).
+Het/de meetpunt(en) gelegen in de deelrastercel(len) met het /de laagste rangnummer(s) zal/zullen dan geselecteerd worden.
+Deze werkwijze laat ook gemakkelijk toe om reserve-punten te kiezen.
+Het zijn de meetpunten die in rang volgen op de laagste rang van een gekozen punt.
+
+#### Samenvatting werkgang selectie peilbuizen
```

Comment:
Ik raad aan om dit stuk te vervroegen, naar net vóór de titel 'Indeling van rastercellen in categorieën', en als volgt in te leiden.

```suggestion
#### Overzicht werkgang selectie peilbuizen

In de volgende paragrafen wordt in detail uiteengezet hoe we komen tot een selectie van peilbuizen per rastercel en GT-groep (en zonodig tot bijkomende nieuwe locaties), uitgaande van de waarden `B` en `G`.
Daarom schetsen we hier eerst een overzicht van deze workflow - zie ook figuur \@ref(fig:workflow-selectiepb-part2).

```